### DISCUSSION TITLE
Sunbird ED Learning series

### DISCUSSION THREAD

**Discussion by luckynvdu** [Sunbird-Ed/Community#45] [General Discussion]
Hello Everybody, As part of Sunbird ED knowledge sharing, we are going to create videos on the following topics in the upcoming quarter to help understand Sunbird ED better

1. What are content and collection? Detailed Explanation of content metadata.
2. Different types of collections (Trackable and Non-Trackable)
3. Form configurations and their role in Sunbird ED?
4. What is a framework? framework tagging for a content/collection?
5. What is a channel or organization? How is a Framework created under an organization?
6. What are the primary categories? Additional categories and tags used for ease of Discovery?
7. UI Widgets that can be configured as part of form configuration?

Request the community members to share their thoughts on additional topics we should cover and also contributions they would like to bring for the same

@sharathkashyap @Krishnaj20 @vinukumar-vs @pallakartheekreddy @pallakartheekreddy @reshmi-nair @rhwarrier @anandp504 @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @SanthoshVasabhaktula @vrayulu @ashokreddy1208 @gandham-santhosh @kameshbhr @mmanimegalai @madhucr @alok-os @pramodkvarma

**Comment by pramodkvarma** [comment]
Fantastic! What about other learning interactions (groups, discussions, events, projects, etc) and also things like credentialing, DIAL, etc.?

**Reply by sharathkashyap** [reply]
@pramodkvarma We will add most of these topics as suggested.

**Comment by aks30** [comment]
@kiranharidas187 , @vishwanath1004 and @5Amogh if you guys want to add any topics

**Comment by vinukumar-vs** [comment]
@luckynvdu @sharathkashyap As part of knowledge-friday sessions, we have covered the below points. I am attaching the presentations here. Check if it helps
[Google drive: All knowledge friday sessions](https://drive.google.com/drive/folders/1Xc-2LpYjbye4qu9IVz_qqHvX_T4qIRn0?usp=sharing)


- ObjectType (Content, Collection etc..)
- Primary Category

We have planned to cover Framework in the following session. For more details [knowlg-Friday's sessions](https://docs.google.com/spreadsheets/d/16W65_qVxMajem4ee-vqRBIfix8u3qMgmbhpIR-TdS5Q/edit?usp=sharing)

For more details of topics, please find the below [Sunbird-Knowlg](https://github.com/orgs/Sunbird-Knowlg/discussions) discussion threads
[Knowlg-Friday's Discussion threads](https://github.com/orgs/Sunbird-Knowlg/discussions?discussions_q=friday)

**Reply by sharathkashyap** [reply]
@vinukumar-vs we have gone through some of these videos. The Explanation has been more from "Inside out" perspective. We are trying to provide "Outside In" perspective for the ecosystem. As there could be multiple adaptors in the future who look at KP from adapting. We would also include the outcomes as part of these explanation.

**Comment by kiranharidas187** [comment]
I would like to know more about the Sunbird-SDK

**Reply by sharathkashyap** [reply]
@kiranharidas187 We will include it as part of Sunbird Mobile SDK

**Comment by kameshbhr** [comment]
@luckynvdu @sharathkashyap This will be of great help! From DIKSHA perspective, following topics are of highest priority.
- Form configurations and their role in Sunbird ED?
- UI Widgets that can be configured as part of form configuration?

We can probably have multiple sessions about configuration such as:
- Config related to Explore tabs in Portal - Home, All and other tabs
- Config related to tabs in Mobile
- Config related Editors
- Config related to backend services
- Config related to Frameworks
- Config related to Assets - asset categories etc.

We can also plan it like an interactive session together with DIKSHA team if that makes sense.
